src/researches/findKeywordInPageTitle.js

```diff
@@ -13,12 +13,28 @@ var escapeRegExp = require( "lodash/escapeRegExp" );
  */
 
 module.exports = function( paper ) {
-	var title = paper.getTitle();
-	var keyword = escapeRegExp( paper.getKeyword() );
-	var locale = paper.getLocale();
-	var result = { matches: 0, position: -1 };
-	result.matches = wordMatch( title, keyword, locale );
-	result.position = title.toLocaleLowerCase().indexOf( keyword );
+	const title = paper.getTitle();
+	// const locale = paper.getLocale();
+
+	const keywordForms = buildKeywordForms( paper );
+
+	let result = { matches: 0, position: -1 };
+	result.matches = matchTextWithArray( title, keywordForms ).length;
+
+	let positions = [];
+	const titleLowerCase = title.toLocaleLowerCase();
+
+	keywordForms.forEach( function( form ) {
+		const formToLowerCase = form.toLocaleLowerCase();
+		const keywordFormIndex = titleLowerCase.indexOf( formToLowerCase );
+
+		if ( keywordFormIndex > -1 ) {
+			positions = positions.concat( keywordFormIndex );
+		}
+		}
```

Comment:
Should have an indentation of one tab less (see `grunt check`)

Reply:
Fixed.